iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/SwitchImpl.java

```diff
@@ -137,4 +139,48 @@ protected String getTypeDescription() {
         return "Switch";
     }
 
+    @Override
```

Comment:
For me, a switch does not have operational limits. It is going to increase considerably our network for nothing because secrutiy analysis are run on bus/breaker view.